resolwe_bio/tools/bamtobigwig.sh

```diff
@@ -8,6 +8,12 @@ bin_size="$3"
 time_limit="$4"
 species="$5"
 
+BAM_SEQUENCES=`samtools view -c "${bam_file}"`
+if [ "${BAM_SEQUENCES}" == 0 ]; then
+  re-warning "Bam file has no entries. No bigWig file will be made."
```

Comment:
I think this should be `re-error`.  It is great to have this Error before starting bamCoverage.